app/controllers/system/admin/instance/users_controller.rb

```diff
@@ -0,0 +1,32 @@
+class System::Admin::Instance::UsersController < System::Admin::Instance::Controller
+  load_and_authorize_resource :instance_user, class: InstanceUser.name,
```

Comment:
Eh, don't you think that `@user.user` is confusing ?  Here I'm actually fetching the instance_users but present `instance_user.user`